### PR TITLE
Move content from zowe.org vNext to Docs

### PR DESCRIPTION
**Pull request by balhar-jakub**
Describe your pull request here:

As agreed on the meeting on 06/28/2024 this is PR that moves the vNext content to the V3 section of the Zowe Docs.

List the file(s) included in this PR:
zowe-v3-conformance-criteria.md, zowe-v3-frequently-asked-questions.md, sidebar.js, zowe-v3-office-hours.md

After creating the PR, follow the instructions in the comments.

### CODE REVIEW

File: docs/whats-new/zowe-v3-frequently-asked-questions.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+## General questions
+
+1. When will Zowe V3 be released?
+  
+   Zowe V3 is scheduled to be released on Sept. 28, 2024. Any changes to the plan will be communicated via all available channels.
+
+2. When will Zowe V3 be available for preview?
+  
+   Currently, Zowe V3 pre-release versions are available for the PAX, Zowe CLI, Zowe Client Node.js SDKs, and Zowe Explorer for Visual Studio Code. **[Intellij? Zowe Desktop?]**
```
Comment (anaxceron) [inline docs/whats-new/zowe-v3-frequently-asked-questions.md:11]: @balhar-jakub - what about IntelliJ and Zowe Desktop?

File: docs/whats-new/zowe-v3-frequently-asked-questions.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+2. With the introduction of the new cloud gateway, how is the configuration going to change?
+
+   The cloud gateway configuration for V3 will move to the currently used gateway configuration space (for example, zowe.components.gateway).The configuration that was Zuul specific will be removed and replaced with the Spring Cloud Gateway configuration.
+
+3. How does client certificate authentication work in Zowe V3?
+
+   Northbound authentication accepts client certificates. The API Mediation Layer then transforms the client certificate to another method of authentication (such as a JWT token, SAF IDT, or passticket) and this new method is then accepted by southbound services. Currently, we do not plan to support authentication with client certificates to z/OSMF.
+
+4. Will you identify deprecated functionality such as passtickets? **[where will this functionality be identified? are passtickets really deprecated? first FAQ says no.]**
```
Comment (anaxceron) [inline docs/whats-new/zowe-v3-frequently-asked-questions.md:29]: @balhar-jakub - please respond

File: docs/whats-new/zowe-v3-frequently-asked-questions.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+In broad terms, yes. However, some of the functionaity that was available in Zowe V2 has been changed (or removed). The changes are listed in these [CLI](https://github.com/zowe/zowe-cli/issues/1694) and [Imperative](https://github.com/zowe/imperative/issues/970) lists of breaking changes. **[use box docs]**
+
+### Zowe Application Framework, ZSS
+
+1. Is it possible to run 32-bit plug-ins at the same time as 64-bit plug-ins with ZSS?
+
+   No. ZSS runs in either 32-bit or 64-bit mode, which means it can accommodate only one type of plug-in.
+
+2. Are **[what kind of?]** libraries updated to the current latest versions?
```
Comment (anaxceron) [inline docs/whats-new/zowe-v3-frequently-asked-questions.md:57]: @balhar-jakub - please respond

File: docs/whats-new/zowe-v3-frequently-asked-questions.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+4. Is the ZEN application running on Zowe Desktop or on z/OS?
+
+ZEN runs on a laptop/desktop and connects to z/OS via standard methods.
+
+5. Is it possible to validate and change the `zowe.yaml` file and job definition within ZEN?
+
+Yes.
+
+6. What happens if in ZEN you click on the SMP/E? **[what is smp/e here? is it a button? a drop-down option? how do users get to smp/e?]**
```
Comment (anaxceron) [inline docs/whats-new/zowe-v3-frequently-asked-questions.md:73]: @balhar-jakub - please respond

File: docs/whats-new/zowe-v3-frequently-asked-questions.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+2. The jump to Java 17 means that we have to maintain two separate versions of our application, one with apiml-enabler V2 and one with V3, to support customers who want to stay with Java 8. Do you have a recommendation or a workaround for supporting both Java versions?
+
+   The plain Java Enabler from Zowe V2 works in Zowe V3. You can keep this until Zowe V4.
+
+   Zowe V4 will only support Spring Boot, which requires Java 17. We recommend building applications for Java 17 to ensure that applications continue to be compatible with API ML.
+
+3. Can you speak about the migration from Zuul to Spring Cloud Gateway? Today there are two separate gateway services in API ML with separate config.
+
+   We expect to support only Spring Cloud Gateway. The spring-cloud-gateway configuration should move under the component.gateway namespace. We expect to have **[should users be doing the config here?]** one Spring Cloud Gateway cluster ahead **[ahead or between?]** of multiple sysplexes and then one cluster on every sysplex. Most of the configuration that was used should remain, including the new one for Spring Cloud Gateway.
```
Comment (anaxceron) [inline docs/whats-new/zowe-v3-frequently-asked-questions.md:107]: @balhar-jakub - please respond

File: docs/whats-new/zowe-v3-frequently-asked-questions.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+
+4. If I have a legacy gateway deployed, how will I migrate to the new gateway? Will the old gateway be removed?
+
+   The old gateway is removed in Zowe V3, but nothing should change from the point of view of the user. However, we may deliver a configuration utility to help with this transition, if needed.
+
+5. How would a client-side end user find and obtain the correct API ML service instance ID for the desired instance of their service?
+
+   In Zowe V2 **[is V2 correct here?]**, clients can use the header instance ID to route communications to a specific instance. Clients can get instance IDs for specific services via an API on the discovery service. We are planning to improve the method for finding service IDs in Zowe V3.
```
Comment (anaxceron) [inline docs/whats-new/zowe-v3-frequently-asked-questions.md:115]: @balhar-jakub - please respond

File: docs/whats-new/zowe-v3-frequently-asked-questions.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+6. Will the LPAR ID be available for the clients to obtain?
+
+   It is not currently available, but we are scheduled to work on this functionality in 2024.
+
+7. API ML static onboarding locates templates that are then used to set variables in the api-defs directory. No manual user action is required. Will this automated process still be available in Zowe V3?
+
+   Static onboarding will continue to be available. The recommendation for Zowe V3 is to move the directory from Zowe workspace **[what is Zowe workspace?]**. The `zowe.yaml` file contains a parameter called `components.discovery.alternativeStaticApiDefinitionsDirectories` that specifies where the directories for static definitions reside.
```
Comment (anaxceron) [inline docs/whats-new/zowe-v3-frequently-asked-questions.md:123]: @balhar-jakub - please respond

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 1 above)
Comment (balhar-jakub) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:11]: I believe that it's now available also for Intellij.

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 2 above)
Comment (balhar-jakub) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:29]: Passtickets will be deprecated with V3, but they will be supported and possibly even through to V4

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 3 above)
Comment (balhar-jakub) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:57]: Angular and Webpack are the libraries.

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 4 above)
Comment (balhar-jakub) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:73]: SMP/E is an option that you can select within the UI, probably a button but @1000TurquoisePogs know more details.

File: docs/whats-new/zowe-v3-frequently-asked-questions.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+
+8. Does the Zowe Store handle Zowe upgrades?
+
+No, it offers server side extensions.
+
+### Zowe System Installation and Configuration
+
+1. Does Zowe provide a Software Bill of Materials (SBOM)?
+
+   Zowe V3 provides SBOMs in an SPDX format. **[have there been changes to the format? if so, what are the changes?]**
```
[the remark this answers is not in the document]
Comment (balhar-jakub) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:89]: I believe there were some changes to the format, but @MarkAckert will know details.

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 5 above)
Comment (balhar-jakub) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:107]: The users shouldn't need to do any specific configuration. 
Ahead of multiple sysplexes. THe diagram is here - https://docs.zowe.org/stable/user-guide/api-mediation/api-mediation-multi-tenancy#overview-of-central-and-domain-api-mls

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 6 above)
Comment (balhar-jakub) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:115]: Yes, it's supported in V2 and will continue to be supported in V3 with improvements to the method.

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 7 above)
Comment (balhar-jakub) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:123]: https://docs.zowe.org/stable/appendix/zowe-glossary#workspace-directory

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (MarkAckert) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:89]: SBOMs are available in the SPDX 2.2 format from the Downloads page. When v3 goes GA, we may choose to upgrade to the SPDX 3.0 format when it becomes available.

File: docs/whats-new/zowe-v3-frequently-asked-questions.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+
+### Zowe Application Framework, ZSS
+
+1. Is it possible to run 32-bit plug-ins at the same time as 64-bit plug-ins with ZSS?
+
+   No. ZSS runs in either 32-bit or 64-bit mode, which means it can accommodate only one type of plug-in.
+
+2. Are **[what kind of?]** libraries updated to the current latest versions?
+
+In Zowe V3, the Application Framework uses Angular 16 and Webpack 5, which are the latest current releases.**[are there other libraries that aren't up-to-date, and if so, should we list them here?]**
```
[the remark this answers is not in the document]
Comment (adam-wolfe) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md]: If we change the question to "Are Angular and Webpack updated to the latest current versions?", I think the existing answer would we adequate.

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 5 above)
Comment (anaxceron) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:107]: Follow-up question: What do we mean by "two separate services"? Are we talking about running both zuul and spring cloud gateway, or are we talking about a configuration w/ a central API ML gateway _and_ multiple domain API ML gateways? The current answer discusses multiple domain gateways behind the central gateway, but it's not clear how this relates back to the FAQ question. Please advise.

File: docs/whats-new/zowe-v3-frequently-asked-questions.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+
+   Yes.
+
+6. What happens if in ZEN you click on the SMP/E? **[what is smp/e here? is it a button? a drop-down option? how do users get to smp/e?]**
+
+   You are guided through the SMP/E installation steps.
+
+7. What is the connection between Zowe Store and ZEN?
+
+   Zowe Store is separate from ZEN, but they do share **[some commands? all commands?]** `zwe` commands.
```
Comment (anaxceron) [inline docs/whats-new/zowe-v3-frequently-asked-questions.md:79]: @balhar-jakub - please respond

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 5 above)
Comment (balhar-jakub) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:109]: We talk about a cluster of central Gateways. In V2 it's always Spring Cloud Gateway with configuration that makes the cluster behave as central. 

I am a bit confused. Original question was about migrating between Zuul and Spring Cloud Gateway. 
This question has two parts answer. 

For domain gateways there should be no change. The properties will remain in gateway namespace. 
For Central gateway there is a change as a result, the properties move from cloud-gateway to gateway namespace.

File: docs/whats-new/zowe-v3-frequently-asked-questions.md [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+
+Yes.
+
+6. What happens if in ZEN you click on the SMP/E? **[what is smp/e here? is it a button? a drop-down option? how do users get to smp/e?]**
+
+You are guided through the SMP/E installation steps.
+
+7. What is the connection between Zowe Store and ZEN?
+
+Zowe Store is separate from ZEN, but they do share **[some commands? all commands?]** `zwe` commands.
```
[the remark this answers is not in the document]
Comment (balhar-jakub) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md]: @1000TurquoisePogs Do you know who has the answer?

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 10 above)
Comment (balhar-jakub) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:81]: Sean Grady knows the answer.

Comment (1000TurquoisePogs) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:81]: I've never heard of the Zowe Store.
But ZEN shouldn't be called ZEN. It's the "Zowe Server Install Wizard".
ZEN is a codename, hence the confusion.

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 4 above)
Comment (1000TurquoisePogs) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:75]: The Zowe Server Install Wizard supports installation of Zowe servers via the PAX or SMPE methods. For more information refer to https://github.com/zowe/docs-site/pull/3663 documentation

File: docs/whats-new/zowe-v3-frequently-asked-questions.md [hunk 12]
```diff
@@ -0,0 +1,125 @@
...
+
+2. Will the APIs that Zowe CLI provides for extenders remain the same?
+
+   In broad terms, yes. However, some of the functionality that was available in Zowe V2 has been changed (or removed). The changes are included in these [CLI](https://ibm.ent.box.com/s/vqu92d82b4wk0i6fupo8glbrxvufn4zw) and [Imperative](https://github.com/zowe/imperative/issues/970) lists of breaking changes.
+
+### Zowe Application Framework, ZSS
+
+1. Is it possible to run 32-bit plug-ins at the same time as 64-bit plug-ins with ZSS?
+
+   No. ZSS runs in either 32-bit or 64-bit mode, which means it can accommodate only one type of plug-in.
```
Comment (1000TurquoisePogs) [inline docs/whats-new/zowe-v3-frequently-asked-questions.md:57]: Strange but true: 31-bit. z/OS is has 31-bit or 64-bit programs. There is no 32-bit.

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 10 above)
Comment (janan07) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:81]: Can we remove this question as it appears this "Zowe Store" is not a current option?  Or is it?

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 4 above)
Comment (janan07) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:75]: This question and response does not provide the reader with any valuable information. The response is self-evident. I suggest the question either be removed or, if there is any additional information about how the wizard assists with the other install methods, for instance, that one particular method is the easist to configure for testing puposes (i.e. convenience build) then the question should be something more like, "What is the most straight-forward method of servier side installation made available through the Zowe Server Install Wizard?". Another valuable question could be, "Are the fields in which the user needs to provide inputs different according to the installation option I choose in the Zowe Server Install Wizard?" This answer would be useful as it explains that the various installation methods present different fields which the user is asked to populate. Isn't it in fact true that some option, perhaps via SMP/E installs the runtime automatically?

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 10 above)
Comment (balhar-jakub) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md]: I would remove the question

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 4 above)
Comment (anaxceron) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md]: I removed the question.

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (anaxceron) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md]: I removed the question b/c no one seems to understand what "Zowe Store" refers to.

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 10 above)
Comment (anaxceron) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md]: I removed the question.

File: docs/whats-new/zowe-v3-frequently-asked-questions.md (hunk 5 above)
Comment (dkelosky) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:109]: Hey Jakub, We have a few questions/comments after discussing with the team in Pitt, can you take a look?

> For domain gateways there should be no change.

Is it correct that for v3, Spring Cloud Gateway will be used for domain gateways without any configuration change? 

> For Central gateway there is a change as a result, the properties move from cloud-gateway to gateway namespace.

Does this mean that config which was previously under `cloud-gateway` in `zowe.yaml` that they config will now be under `gateway`? 

> We expect to support only Spring Cloud Gateway. The spring-cloud-gateway configuration should move under the component.gateway namespace. We expect to have **[should users be doing the config here?]** one Spring Cloud Gateway cluster ahead **[ahead or between?]** of multiple sysplexes and then one cluster on every sysplex. Most of the configuration that was used should remain, including the new one for Spring Cloud Gateway.

For this, we should probably remove "should" in favor of something more definitive unless we truly don't know the outcome.  For example, "... configuration should move ... " to "...configuration **will** move ..."

Thanks!

Comment (balhar-jakub) [reply docs/whats-new/zowe-v3-frequently-asked-questions.md:109]: I believe that now we can move to more definitive answer, this was move of questions from Office Hours in January. 

Spring Cloud Gateway will be used for domain gateways without any configuration change. 
Spring Cloud Gateway in central will move to gateway as it's the same component in a different configuration.

**Review by anaxceron — APPROVED** [review]

**Pull request merged**